.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,73 @@
+name: 🐞 Bug
```

Comment:
Cute 🐞 lady bug kk

In this repository, we already have a mark-down template.
So, It seems good to keep the existing name as it is.
🐞 Bug -> 🐞 Bug report